cfme/tests/automate/custom_button/test_buttons.py

```diff
@@ -686,3 +698,44 @@ def test_simulated_object_copy_on_button(appliance, provider, setup_provider, bu
     assert view.advanced.system.read() == "Automation"
     assert view.advanced.message.read() == "test_bz"
     assert view.advanced.request.read() == "InspectMe"
+
+
+@pytest.mark.tier(1)
+@pytest.mark.meta(blockers=[BZ(1755229)], automates=[1755229])
+def test_under_group_multiple_button_crud(appliance, button_group, dialog):
+    """Test multiple button deletion under same group
+
+    Bugzilla:
+        1755229
+
+    Polarion:
+        assignee: ndhandre
+        initialEstimate: 1/10h
+        caseimportance: critical
+        startsin: 5.8
+        casecomponent: CustomButton
+        tags: custom_button
+        testSteps:
+            1. Create a Button Group
+            2. Create button and delete button multiple time
```

Comment:
Suggestion:
```
2. Create button and delete it
3. Repeat step 2 multiple times
```

Reply:
Also please add expected result.

Reply:
@kedark3  for automated test `ExpectedResult` not required.

Reply:
Got it. Is that documented somewhere?